### PR TITLE
Packaging work

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: atomic-pkglayer.spec [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Summary: A tool to install debug packages on Atomic Host
+Name: atomic-pkglayer
+Version: 2015.3
```
Comment (miabbott) [inline atomic-pkglayer.spec:3]: Latest tag/release is 2015.4

File: Makefile.dist-packaging [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	done; \
+	mv $(PKG_VER).tar{.tmp,}; \
+	rm -f $(PKG_VER).tar.xz; \
+	xz $(PKG_VER).tar 
+
+srpm: dist-snapshot
+	(srcdir=$$(pwd); tmpdir=$$(mktemp -d); \
+	 sed -e "s,^Version:.*,Version: $(GITREV_FOR_PKG)," $(PACKAGE).spec > $${tmpdir}/$(PACKAGE).spec; \
+	 cp $(PKG_VER).tar.xz $${tmpdir}; \
+	 cd $${tmpdir} && ${srcdir}/rpmbuild-cwd -bs $(PACKAGE).spec)
```
Comment (miabbott) [inline Makefile.dist-packaging]: Makefiles are not my thing, but...

Should the resulting `atomic-pkglayer*src.rpm` get copied back to `$srcdir`?

When I ran `make -f Makefile.dist-packaging rpm`, I saw the following:

```
 cd ${tmpdir} && /home/miabbott/workspaces/atomic-pkglayer/rpmbuild-cwd -bs atomic-pkglayer.spec)
Wrote: /tmp/tmp.zYhUubnT0H/atomic-pkglayer-2015.3.10.g3935111-2.fc23.src.rpm
/home/miabbott/workspaces/atomic-pkglayer/rpmbuild-cwd --rebuild atomic-pkglayer-2015.3.10.g3935111*.src.rpm
error: cannot open atomic-pkglayer-2015.3.10.g3935111*.src.rpm: No such file or directory
Makefile.dist-packaging:35: recipe for target 'rpm' failed
make: *** [rpm] Error 1
```

Comment (cgwalters) [inline Makefile.dist-packaging]: Looks like I copied a WIP version.  Fixed.

File: atomic-pkglayer.spec (hunk 1 above)
Comment (cgwalters) [inline atomic-pkglayer.spec:3]: Looks like I fucked up the tag =(  It's not pointing to a commit reachable from master.  Sigh...will fix it with v2016.1.

**Pull request merged**